pkg/proxy/ipvs/proxier.go

```diff
@@ -157,6 +160,12 @@ type Proxier struct {
 	nodePortSetTCP *IPSet
 	// nodePortSetTCP is the bitmap:port type ipset where stores all UDP node port
 	nodePortSetUDP *IPSet
+	// lbIngressLocalSet is the hash:ip type ipset where stores all service ip which with externaltrafficPolicy=local
```

Comment:
"lbIngressLocalSet is the hash:ip type ipset which stores all service ip's with externaltrafficPolicy=local